Description: Fix path of the libnautilus-extension with multi-arch
Author: Laurent Bigonville <bigon@debian.org>

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -417,8 +417,6 @@
   PKG_CHECK_MODULES([NAUTILUS],[gtk+-3.0 $MM gthread-2.0 libnautilus-extension >= $NAUTILUS_REQUIRED],
                     [],[AC_MSG_ERROR([libnautilus-extension not found; use --disable-nautilus to disable the nautilus extensions])])
   NAUTILUS_EXTENSION_DIR=`$PKG_CONFIG --variable=extensiondir libnautilus-extension`
-  NAUTILUS_EXTENSION_DIR=${NAUTILUS_EXTENSION_DIR#`$PKG_CONFIG --variable=libdir libnautilus-extension`}
-  NAUTILUS_EXTENSION_DIR=${NAUTILUS_EXTENSION_DIR#/}
   AC_SUBST(NAUTILUS_EXTENSION_DIR)
   AC_SUBST(NAUTILUS_CFLAGS)
   AC_SUBST(NAUTILUS_LIBS)
Index: b/properties/Makefile.am
===================================================================
--- a/properties/Makefile.am
+++ b/properties/Makefile.am
@@ -15,7 +15,7 @@
 
 if ENABLE_NAUTILUS
 
-nautilusextensiondir=$(libdir)/$(NAUTILUS_EXTENSION_DIR)
+nautilusextensiondir=$(NAUTILUS_EXTENSION_DIR)
 nautilusextension_LTLIBRARIES = libevince-properties-page.la
 
 libevince_properties_page_la_SOURCES = 	\
